.github/workflows/docker.yml

```diff
@@ -1,7 +1,7 @@
 name: ci
 
 on:
-  [push, pull_request]
```

Comment:
since we tend to push to this repo for pull requests, avoids having redundant jobs